starfish/test/full_pipelines/api/test_merfish.py

```diff
@@ -274,4 +270,4 @@ def test_merfish_pipeline_cropped_data():
 
     corrcoef = np.corrcoef(tmp[:, 1], tmp[:, 0])[0, 1]
 
-    assert np.round(corrcoef, 4) == 0.9739
+    assert np.round(corrcoef, 4) == 0.894
```

Comment:
this seems like a pretty big change.  @dganguli / @ambrosejcarr this ok?

Reply:
well, did floats get merged? if so, this should be re-based off of master, in which case the the jump is from 0.93 -> 0.894 instead of 0.9739 -> 0.894. which is less bad?

regardless, i would leave the test_merfish.py as is. But NOT commit the notebook (or it's .py representation). This has the big con that the test diverges from the notebook. But that's OK I think. The point of the test is to test the CLI -- which this does. 

The real solution, is to implement a ScaleByValues filter, which makes a PipelineComponent that achieves L115-L130 of the .py representation of the notebook. Then, the test would also use this filter. Then the test and the notebook would be in sync. 

But since we don't have that solution, I think we just make an Issue, nuke the .ipynb (and it's .py version) and merge. Thoughts?

Reply:
> well, did floats get merged? if so, this should be re-based off of master, in which case the the jump is from 0.93 -> 0.894 instead of 0.9739 -> 0.894. which is less bad?

This is actually the correct correlation -- this test works off a smaller part of one of the images, and because this subset is _not_ affected negatively by deconvolution (which has per-_image_ funkiness), it does not change, and is actually 0.97. Does that make sense @dganguli ?

Reply:
This is based off master, so this is actually the api test. I thought we were going to try and unify the api tests and the notebooks by calling the notebook in the api test instead of going though all the steps. If that's the case I can just remove the filter from this api test and notebook and only add it to the cli test in my other branch